### PR TITLE
blog: meet our contributors APAC Aus-Nz region

### PR DESCRIPTION
**Pull request by Debanitrkl**
This PR adds the second blog post for APAC region(AUS-NZ), in the series of ["Meet Our Contributors - APAC"](https://docs.google.com/document/d/1zB-FOvfbnKbiWlGgInnDB06hDfLsTQuuKsXQCqc-lL0/edit#heading=h.yr3qa2ko7xq) blog series.

Reference issue: https://github.com/kubernetes/community/issues/4469#issuecomment-942422348

### CODE REVIEW

File: content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+---
+
+Good day, everyone :wave:
+
+Welcome back to the second episode of the "Meet Our Contributors" blog post series for APAC.
+
+This post will feature four outstanding contributors from the Australia and New Zealand regions, who have played diverse leadership and community roles in the Upstream Kubernetes project.
+
+So, without further ado, let's get straight to the blog.
```
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md:18]: (nit)
```suggestion
So, without further ado, let's get straight to the article.
```
(“blog post” and “blog” are also OK). A blog is a series of articles.

File: content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+His contribution path was notable in that he began working on major areas of the Kubernetes project early on, skewing his trajectory.
+
+He asserts the best thing a new contributor can do is to "start contributing". Naturally, if it is relevant to their employment, that is excellent; however, investing non-work time in contributing can pay off in the long run in terms of work. He believes that new contributors, particularly those who are currently Kubernetes users, should be encouraged to participate in higher-level project discussions. 
+
+> _Just being active and contributing will get you a long way. Once you've been active for a while, you'll find that you're able to answer questions, which will mean you're asked questions, and before you know it you are an expert._
+
+---
+
+If you have any recommendations/suggestions for who we should interview next, please let us know in #sig-contribex. Your suggestions would be much appreciated. We're thrilled to have additional folks assisting us in reaching out to even more wonderful individuals of the community.
```
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md:68]: Maybe link to https://kubernetes.slack.com/messages/sig-contribex ?

**Review by sftim — COMMENTED** [review]
Small nits

**Comment by sftim** [comment]
@Debanitrkl is there a similar PR (to mirror this one) opened against [k/website](https://github.com/kubernetes/website/)?

File: content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (Aus-NZ region)"
+date: 2022-03-02
```
Comment (sftim) [inline content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md:4]: This date is now in the past

**Review by sftim — CHANGES_REQUESTED** [review]
We should set a publication date that's in the future, and aim to merge on that date.

File: content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md (hunk 3 above)
Comment (jberkus) [reply content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md:4]: @Debanitrkl lemme know what date you want to publish this.

**Comment by Debanitrkl** [comment]
> @Debanitrkl is there a similar PR (to mirror this one) opened against [k/website](https://github.com/kubernetes/website/)?

Not yet I was waiting for this PR to get merged and then get a PR up at k/website

File: content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md (hunk 3 above)
Comment (Debanitrkl) [reply content/en/blog/2022/meet-our-contributors-APAC-au-nz-region-02.md]: Updated the date

**Comment by jberkus** [comment]
Looks good.
/lgtm
/approve

**Pull request merged**